IrisSoftware/main.py

```diff
@@ -6,9 +6,10 @@
 from PySide6 import QtCore
 import cv2
 import pyautogui
-from detectEyes import detectEyes, DetectionType
-from computeScreenCoords import computeScreenCoords
-from ui.UI import UI, CALIBRATION_FILE_NAME
+from IrisSoftware.detectEyes import detectEyes, DetectionType
+from IrisSoftware.computeScreenCoords import computeScreenCoords
+from IrisSoftware.UI import UI, CALIBRATION_FILE_NAME
+from IrisSoftware.Camera import Camera
```

Comment:
If everything is in the same `IrisSoftware` folder, do these imports need to be prepended with the folder name?

Reply:
> Looks good! I left a few comments about simplifying the project structure. And what is the `__init__.py` file for? If it's just blank, should we delete it?

`__init__.py` declares a directory as a Python module (https://stackoverflow.com/questions/448271/what-is-init-py-for). Not sure if we need it, but was just recommended by the `poetry` instructions.

Reply:
> If everything is in the same `IrisSoftware` folder, do these imports need to be prepended with the folder name?

I think this has to do with the inclusion of the `__init__.py`. With it, just importing `detectEyes` throws an error. So I guess that's what that file does behind the scenes? Not sure if we need `__init__.py` tbh and we could just swap back

Reply:
In that stack overflow post you linked, it says that `__init__.py` is no longer needed in python versions greater than 3.3, so we can go without it here.